resources/views/media/manager.blade.php

```diff
@@ -55,6 +55,11 @@
             <i class="voyager-refresh"></i>
         </button>
         <div class="btn-group offset-right">
+            <button type="button" class="btn btn-default" id="addAll" v-if="allowUpload" v-on:click="addAllFilesToInput();" 
```

Comment:
Thanks for your PR.
We need to add a check if the media-manager is used as a media-picker.
Otherwise it would show the button when you normally browse media and might fail if you click it.
You can simply add `v-if="hidden_element"`